nevergrad/functions/corefuncs.py

```diff
@@ -311,6 +311,21 @@ def lunacek(x: np.ndarray) -> float:
 # following functions using discretization should not be used with translation/rotation
 
 
+@registry.register_with_info(no_transform=True)
+def discrete_onemax(y: np.ndarray) -> float:
```

Comment:
this is just renaming the other functions, and you're not using it in ArtificialFunction so no need to register
so instead of this you should just rename `_onemax` to `onemax` and so on (maybe with some adaptation to convert arrays into list?), and use them directly in the xps . Or is there something I'm missing?

Reply:
Actually there is a difference, namely there is no discretization, we work directly on {0,1,2,3...,A-1}. Do you still think we should use _onemax ?

Reply:
Done!